pkg/scheduler/apis/config/v1beta1/conversion.go

```diff
@@ -46,6 +46,149 @@ func getPluginArgConversionScheme() *runtime.Scheme {
 	return pluginArgConversionScheme
 }
 
+func Convert_v1beta1_Plugins_To_config_Plugins(in *v1beta1.Plugins, out *config.Plugins, s conversion.Scope) error {
+	if err := convertToInternalPluginSet(in.QueueSort, &out.QueueSort, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PreFilter, &out.PreFilter, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Filter, &out.Filter, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PostFilter, &out.PostFilter, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PreScore, &out.PreScore, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Score, &out.Score, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Reserve, &out.Reserve, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Permit, &out.Permit, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PreBind, &out.PreBind, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.Bind, &out.Bind, s); err != nil {
+		return err
+	}
+	if err := convertToInternalPluginSet(in.PostBind, &out.PostBind, s); err != nil {
+		return err
+	}
+	return nil
+}
+
+func convertToInternalPluginSet(in *v1beta1.PluginSet, out *config.PluginSet, s conversion.Scope) error {
```

Comment:
Can you use `Convert_v1beta1_PluginSet_To_config_PluginSet` instead?

Reply:
done.